seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/resources/kafka/kafkasource_group_offset_to_console_with_commit_offset.conf

```diff
@@ -63,11 +66,11 @@ sink {
 
               {
                 rule_type = MIN
-                rule_value = 100
+                rule_value = 0
               },
               {
                 rule_type = MAX
-                rule_value = 149
+                rule_value = 99
```

Comment:
Any reason for change test case config?

Reply:
The original test case did not play a role because he did not read the data

Reply:
Maybe there is no suitable comment. The purpose of this test case is not to read any data and ensure that the offset can still be submitted to Kafka normally. Please refer https://github.com/apache/seatunnel/pull/7312
cc @hailin0